src/AccessibilityInsights.Rules/Library/BoundingRectangleContainedInParent.cs

```diff
@@ -70,10 +72,15 @@ private static bool IsBoundingRectangleContained(IA11yElement container, IA11yEl
         protected override Condition CreateCondition()
         {
             // Windows can be any size, regardless of their parents
+            // Certain chrome panes are treated as windows because that's how they behave.
```

Comment:
is this issue only for chrome? IMO, it can happen to any application. if then, how about not considering this case for Menu than excluding Chrome case?

Reply:
This fix is specifically for Chrome, based on an issue we got from a user. As for making this more widely applicable to all menus, I feel like we just don't have enough info right now to justify making that change. I think it would be just as likely to cause an issue as to solve one. Also, if we relax the requirements, that means any cases where a menu should (for some reason) remain in a window would never get called out and therefore be invisible forever. In short, I would like to wait for more data.

Reply:
I disagree with you on the resolution. I'll share one other case which I can think. Let's talk about this more.